test/plugin/constants.rb

```diff
@@ -441,6 +450,14 @@ module Constants
       )
     )
   ).freeze
+  K8S_CONTAINER_PARAMS_CUSTOM = K8S_CONTAINER_PARAMS.merge(
```

Comment:
Where is this actually being used?

Reply:
For whatever reason, searching the page previously only showed one instance of this variable. I see it now.

Reply:
It was a separate file — the diff could have been collapsed by GitHub.